src/BogusTest/DataGeneratorRepeaterBackroundTask.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BogusTest;
+
+public class DataGeneratorRepeaterBackroundTask : IDataGeneratorRepeaterBackroundTask
+{
+    private Task? _timerTask;
+    private readonly PeriodicTimer _timer;
+    private readonly CancellationTokenSource _cts = new();
+    private readonly IDataGeneratorTest _dataGenerator;
+
+
+    public DataGeneratorRepeaterBackroundTask(TimeSpan interval, IDataGeneratorTest dataGenerator)
+    {
+        _timer = new PeriodicTimer(interval);
+        _dataGenerator = dataGenerator;
+    }
+
+    public void Start()
+    {
+        _timerTask = LoadFakeDataEverySecondAsync();
+    }
+
+    private async Task LoadFakeDataEverySecondAsync()
+    {
+        try
+        {
+            while (await _timer.WaitForNextTickAsync(_cts.Token))
+            {
+                _dataGenerator.LoadFakeData();
+            }
+        }
+        catch (OperationCanceledException) { }
```

Comment:
`catch` should never, ever be empty. If you don't know what to do there, at least log an exception